integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -97,12 +99,7 @@ func TestVRFv2Plus(t *testing.T) {
 			vrfv2PlusData,
 			subID,
 			isNativeBilling,
-			*configCopy.VRFv2Plus.General.MinimumConfirmations,
-			*configCopy.VRFv2Plus.General.CallbackGasLimit,
-			*configCopy.VRFv2Plus.General.NumberOfWords,
-			*configCopy.VRFv2Plus.General.RandomnessRequestCountPerRequest,
-			*configCopy.VRFv2Plus.General.RandomnessRequestCountPerRequestDeviation,
-			configCopy.VRFv2Plus.General.RandomWordsFulfilledEventTimeout.Duration,
+			configCopy.VRFv2Plus.General,
```

Comment:
I want your opinion on this @kidambisrinivas 

I dont like this approach of providing just a config into a method, it should be a struct specifically intended for this method.

So one struct could be for consumer.requestRandomness method parameters

```
type RequestRandomnessParameters{
			vrfKeyData.KeyHash,
			subID,
			minimumConfirmations,
			callbackGasLimit,
			isNativeBilling,
			numberOfWords,
			randomnessRequestCountPerRequest,
}
```

Reply:
could you please apply this also for other places, like loggers, wrapper requests

Reply:
I will need to do this anyway, but maybe if its not too of hassle - change the same for VRF v2. Fine if you will not do it

Reply:
I think the config here is fine, since it has the basic list of config parameters for related use cases. At this point, specialising into RequestRandomnessParameters, DirectRequestRandomnessParameters is probably an overkill. 

Config is nice, in the sense it has VRF job specific config parameters used in different places.

Reply:
Based on offline discussion, config seems to be a good fit here, as it is generic enough for different use cases and has all VRF job specific parameters for any additional uses needed in tests